citar.el

```diff
@@ -706,7 +708,31 @@ If FORCE-REBUILD-CACHE is t, force reload the cache."
 
 (defun citar--extract-keys (keys-entries)
   "Extract list of keys from KEYS-ENTRIES alist."
-  (seq-map #'car keys-entries))
+  (seq-map (lambda (key-entry)
+             (if (consp key-entry) (car key-entry) key-entry))
+           keys-entries))
+
+(defun citar--ensure-entries (keys-entries)
```

Comment:
```suggestion
(defun citar--ensure-entries (key-entry-alist)
```

Right?

There are a few remaining examples.

Reply:
No, the argument to `citar--ensure-entries` can contain both KEY and (KEY . VALUE) items, so it should in fact be called `keys-entries`. The return value would be a `key-entry-alist` if it had a name. I think the same is true of all the other arguments that are called `keys-entries`, but it's possible I missed some instances. There are only a very few places where there actually is a `key-entry-alist` variable.